ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -204,11 +210,8 @@ const TierConfig = (props: TierConfigProps) => {
             <AddressTextArea
               accessTierShortName={accessTierShortName}
               emailAddresses={emailAddresses}
-              onBlur={validateTierAddresses}
+              onBlur={filterEmptyAddresses}
```

Comment:
validating `onBlur` was causing the awkward UI.  We no longer do so.  Now validation is limited to the save button, via `validate.js`